lua/plenary/log.lua

```diff
@@ -160,7 +161,7 @@ log.new = function(config, standalone)
           table.insert(inspected, vim.inspect(v))
         end
         return string.format(fmt, unpack(inspected))
-      end)
+      end, unpack(args))
```

Comment:
Can we actually just do `...` here instead of creating and unpacking the table?

Reply:
Fixed and works.